src/Surface.h

```diff
@@ -40,12 +40,15 @@ class CJNISurface : public CJNIBase
   int         describeContents();
   static void PopulateStaticFields();
   void setFrameRate (float frameRate, int compatibility);
+  void setFrameRate(float frameRate, int compatibility, int changeFrameRateStrategy);
```

Comment:
How about not adding another override for >= 31, but to extend the existing function with the new parameter, ignoring it (or logging a warning that the param is ignored) if called with <31? Otherwise you must add version checking code on every user case on callers side.

Reply:
Yeah makes sense, will change it.